shanoir-ng-studies/src/main/java/org/shanoir/ng/study/repository/StudyRepositoryImpl.java

```diff
@@ -43,21 +43,23 @@ public List<StudyStatisticsDTO> queryStudyStatistics(Long studyId) throws Except
 			studyId = (Long) row[0];
 			Long centerId = (Long) row[1];
 			String centerName = (String) row[2];
-			Long subjectId = (Long) row[3];
-			String commonName = (String) row[4];
-			Long examinationId = (Long) row[5];
-			Date examinationDate = (Date) row[6];
-			Long datasetAcquisitionId = (Long) row[7];
-			Date importDate = (Date) row[8];
-			Long datasetId = (Long) row[9];
-			String modality = (String) row[10];
-			String quality = (String) row[11];
+			String centerPrefix = (String) row[3];
```

Comment:
I think you should directly set the DTO properties here, would be more compact and readable :
 ``` java
StudyStatisticsDTO dto = new StudyStatisticsDTO();
 dto.setStudyId((Long) row[0]);
dto.setCenterId((Long) row[1]);
dto.setCenterName((String) row[2]);
...
 ```